ui.tests/test-module/specs/panelcontainer/panelcontainer.authoring.spec.js

```diff
@@ -94,6 +103,34 @@ describe('Page - Authoring', function () {
 
         it('open edit dialog of Panel', function () {
             testPanelBehaviour(panelContainerPathSelector, panelEditPath);
+        });
+
+        it('test data-placeholder-text of Panel', function () {
+            dropPanelInContainer();
+            cy.openEditableToolbar(sitesSelectors.overlays.overlay.component + panelContainerPathSelector);
+            cy.invokeEditableAction("[data-action='CONFIGURE']");
+            cy.get('input[name="./name"]').invoke('val', 'customPanel');
+            cy.get('input[name="./jcr:title"]').invoke('val', 'Custom Panel');
+            cy.get('.cq-dialog-submit').click().then(() => {
+                cy.get('.cq-dialog-submit').should('not.exist');
+            });
+            getPreviewIframeBody().find('.cmp-container__label').should('have.text', 'Custom Panel')
+            getPreviewIframeBody().find("div[data-cmp-adaptiveformcontainer-path='/content/forms/af/core-components-it/blank/jcr:content/guideContainer']").should('have.attr', 'data-placeholder-text', 'Please drag Custom Panel components here');
```

Comment:
why do we need to call getPreviewIframeBody() function for both lines 117 and 118? Can't we just use cy.get() ?

Reply:
Same reason as below.